portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -308,110 +308,128 @@ definition {
+
+			WebContent.viewTitle(webContentTitle = "WC Title");
+
+			WebContent.deleteCP(
+				siteURLKey = "test-site-name",
+				webContentTitle = "WC Title");
+		}
 	}
 
 	@description = "Validade if a Web Content associated with a Display Page Template is shown"
 	@priority = "5"
-	test ContentDashboardExhibitWebContentAssociatedDisplayPageTemplate {
-		JSONWebcontent.addWebContent(
-			content = "WC WebContent Content",
-			groupName = "Test Site Name",
-			title = "WebContent Title");
+	test ShouldExhibitWebContentAssociatedWithDisplayPage {
+		task ("Add a site via JSON") {
+			JSONGroup.addGroup(groupName = "Test Site Name");
+		}
 
-		JSONLayoutpagetemplate.addDisplayPageTemplateEntry(
-			contentType = "Web Content Article",
-			displayPageTemplateEntryName = "Display Page Name",
-			groupName = "Test Site Name",
-			subType = "Basic Web Content");
+		task ("Create and Publish a Display Page Template with Fragment"){
+			JSONLayoutpagetemplate.addDisplayPageTemplateEntry(
+				contentType = "Web Content Article",
+				displayPageTemplateEntryName = "Display Page Name",
+				groupName = "Test Site Name",
+				subType = "Basic Web Content");
+			
+			DisplayPageTemplatesAdmin.openDisplayPagesAdmin(siteURLKey = "test-site-name");
 
-		DisplayPageTemplatesAdmin.openDisplayPagesAdmin(siteURLKey = "test-site-name");
+			DisplayPageTemplatesAdmin.gotoDisplayPage(displayPageName = "Display Page Name");
 
-		DisplayPageTemplatesAdmin.gotoDisplayPage(displayPageName = "Display Page Name");
+			Button.clickPublish();
+		}
 
-		PageEditor.addFragment(
-			collectionName = "Content Display",
-			fragmentName = "Content Display");
+		task ("Create a Web Content and set a Display Page"){
+			JSONWebcontent.addWebContent(
+				content = "Webcontent Content",
+				groupName = "Test Site Name",
+				title = "Webcontent Title");
 
-		PageEditor.editContentDisplay(
-			fragmentName = "Content Display",
-			webcontent = "true",
-			webContentContent = "WC WebContent Content",
-			webContentTitle = "WebContent Title");
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
 
-		PageEditor.clickPublish();
+			WebContentNavigator.gotoEditCP(webContentTitle = "Webcontent Title");
 
-		ApplicationsMenu.gotoPortlet(
-			category = "Content",
-			panel = "Applications",
-			portlet = "Content Dashboard");
+			WebContent.editDisplayPage(
+				pageName = "Display Page Name",
+				pageType = "Display Page Templates");
 
-		AssertTextPresent(
-			locator1 = "ContentDashboard#FIRST_ELEMENT_LISTED",
-			value1 = "WebContent Title");
+			PortletEntry.publish();
+		}
+
+		task ("Check in Content Dashboard if WC is assiciated with Display Page Template") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Content",
+				panel = "Applications",
+				portlet = "Content Dashboard");
+			
+			AssertElementPresent(locator1 = "//span[contains(@title, 'This content has a display page')]");
```

Comment:
You cannot put a path within an assert. You have to create the path element for the Display Page icon in Content dashboard.

Reply:
You should validate that the Display Page associated icon is not present for the WC created. A way to do this is validate that just the WC is listed in the table and the Display Page associated icon is not present.

Reply:
In this case, we want to assert that the icon is being presented.